submissions/Sergey-Kali/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/index.css">
+    <title>My menu</title>
+</head>
+<body>
+   <nav class="nav-panel">
+    <ul class="nav-list">
+        <li class="list-item"><a href="#" class="mail">Mail</a></li>
+        <li class="list-item"><a href="#" class="img">Images</a></li>
+        <li class="list-item">
+            <div class="popup-wrapper">
+                <input type="checkbox" id="popup-button" class="popup-input">
+                <label for="popup-button" class="popup-label"></label>
```

Comment:
Your menu icon is not accessible from the keyboard, please fix it. [Watch video](https://share.vidyard.com/watch/A3JE2XUyGKq6SjdEyNrAt8?)